src/MarginTrading.Backend.Services/AssetPairs/DayOffSettingsService.cs

```diff
@@ -111,7 +111,7 @@ public void Start()
                              dayOffStartDay: DayOfWeek.Friday,
                              dayOffStartTime: new TimeSpan(20, 55, 0),
                              dayOffEndDay: DayOfWeek.Sunday,
-                             dayOffEndTime: new TimeSpan(22, 55, 0),
+                             dayOffEndTime: new TimeSpan(22, 05, 0),
                              assetPairsWithoutDayOff: new[] {"BTCUSD"}.ToHashSet(),
                              pendingOrdersCutOff: new TimeSpan(0, 55, 0)));
```

Comment:
Maybe it would be better to move schedule config to settings.

Reply:
Already moved, these are the default ones